docs/changelog.rst

```diff
@@ -1,5 +1,53 @@
 Changelog
 ================================
+CBC SDK 1.5.0 - Released (TBD)
+--------------------------------------
+
+**Breaking Changes:**
+
+* Alerts V7: This is a new version of the Alerts API with certain changes that are not compatible with code written
+  to the old V6 API. For details, please see the :ref:`Alert Migration Guide <alert-migration-guide>`.
+  New features in Alerts include:
+
+  * Extended alert schema with additional metadata such as process command line and username, parent and child process
+    information, netconn data, additional device fields, MITRE categorization when available, and more
+  * Easier management and consumption of grouped alerts
+  * Ability to mark alerts as “In Progress”
+  * Ability to mark alerts as True Positive or False Positive
+  * Additional fields available for both searching and faceting
+  * Enhanced note management with the ability to add notes to both individual alerts and threats (alerts grouped
+    by threat)
+  * Observed Alerts have been removed from the Alerts API as these events are not considered actionable threats. They
+    can now be retrieved via the Observations API.
+
+* Official support for Python 3.7 has been dropped, since that version is now end-of-life.  Added explicit testing
+  support for Python version 3.12.  **N.B.:** End users should update their Python version to 3.8.x or greater.
+
+New Features:
+
+* External Devices: Added External Device Export and External Device Approvals Export.
+
+Updates:
+
+* Audit log requests have moved from ``CBCloudAPI`` into their own function entry point in the ``platform`` package.
+  The old function has been deprecated.
+* Process search validation has been changed to use the V2 ``POST`` API rather than the old V1 ``GET`` API.
+* ``CBCloudAPI.get_notifications()`` and ``CBCloudAPI.notification_listener()`` have been marked as deprecated.
+
+Documentation:
+
+* Added example script to poll for audit logs.
+* ``CBCloudAPI`` documentation has been pulled out into its own page.
+* Authentication, Getting Started, and Guides pages have been updated.
+* Concepts page has been removed, and the information it contained has moved to other pages.
+* New Searching guide added.
```

Comment:
A link to the search guide would be cool.
https://carbon-black-cloud-python-sdk.readthedocs.io/en/latest/searching/